packages/web-components/src/mixins/focusable.css

```diff
@@ -1,7 +1,7 @@
 /* From the USWDS styles: */
 /*   https://github.com/uswds/uswds/blob/3dc296ec56cd621fe52d918701fd94621d96a198/src/stylesheets/core/mixins/_focus.scss#L12-L13 */
 button:not([disabled]):focus,
-button:not([disabled]):active,
+:host(:not([uswds])) button:not([disabled]):active,
```

Comment:
The USWDS version does not have the focus halo for active states so I have removed it from the uswds variation but I'm confirming if that's what we want too. If we want the active state to have the gold focus, I'll revert this.

Reply:
@jamigibbs I think we want to mimic the style of the USWDS button as much as possible, so it's OK to remove the focus halo from the active states.